libraries/AP_Scripting/AP_Scripting.cpp

```diff
@@ -71,6 +71,11 @@ const AP_Param::GroupInfo AP_Scripting::var_info[] = {
     // @RebootRequired: True
     AP_GROUPINFO("HEAP_SIZE", 3, AP_Scripting, _script_heap_size, SCRIPTING_HEAP_SIZE),
 
+    // @Param: DEBUG_LVL
+    // @DisplayName: Enable Scripting Debug
+    // @Description: Enable scripting debug.
+    // @User: Advanced
```

Comment:
A range and increment whould be helpfull.

Reply:
Oops.  That patch went in in a different PR.  PRs welcome to fix it in master :-)

If you really think every single parameter deserves those attributes modifications to the code being merged in this PR could relatively easily ensure it.